jetstream/tests/test_probe_sets.py

```diff
@@ -32,7 +32,7 @@
     "probes": [
       {
         "kind": "event",
-        "name": "Test Probe",
+        "name": "test-probe-2",
```

Comment:
Events shouldn't have names, I think

Reply:
Are the probe slugs in [the ADR](https://github.com/mozilla/jetstream/blob/main/docs/adr-0001-publishing_metric_docs.md#metadatajson-for-each-experiment-save-some-json-to-gcs) example referring to `name`? Or is that a different slug?

Reply:
Probesets have names, and probesets produce metrics; the probes are not named.

Reply:
(The documentation here isn't about which probes are contained by the probeset; Experimenter knows that. The documentation is about what metrics they generated.)

Reply:
Ah, I see. Removed `name` from the event probe and added the metric slugs to the `probesets` metadata

Reply:
nit: this fixture is still wrong, though :)
```suggestion
```